src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -610,10 +611,11 @@ export class BeneficiariesService {
 		// return this.hasuraService.delete(this.table, { id: +id });
 	}
 
-	public async statusUpdate(req: any) {
+	public async statusUpdate(req: any, header: any) {
```

Comment:
from controller you are passing body, request
where is header coming from?

Reply:
from controller now we are passing body and request. 
In service we are getting body and request.